### PR TITLE
Basic html for landing page and home page

### PR DESCRIPTION
**Pull request by Hekaiyo**
Added front-end HTML/CSS/Assets for the landing page and the home page.

To facilitate demonstrations, I also implemented a very basic controller and routing.

### CODE REVIEW

File: app/views/placeholder/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+<% if @showQuizWarning %>
+<div class="jumbotron text-center" style="background-color: #FFAC6D">
+  <h5>Please take the quiz to see organization matches.</h5>
+</div>
+<% end %>
+<div class="container" style="margin-top:30px">
+
+
+  
+  <div class="row">
```
Comment (juanxcerv) [inline app/views/placeholder/home.html.erb:53]: This is good for now, once we get the real organizations we'll refactor this to be a loop through the matched organizations and maybe add something like pagination if we have too many.

File: app/views/placeholder/landing.html.erb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link href='https://fonts.googleapis.com/css?family=Roboto Slab' rel='stylesheet'>
+    <style>
```
Comment (juanxcerv) [inline app/views/placeholder/landing.html.erb:11]: We might want to make separate css files for conventions sake, just to make it easier to change in the future.

File: app/views/placeholder/landing.html.erb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  <body>
+  
+  
+
+
+<div class="vertical-center" >
+  <div class="container rounded" style="background-color: #FFFFFF; max-width:1000px" >
+    <div class="row">
+      <div class="col-sm-12" style="text-align:center">
+          <%=image_tag("LEAD logo.png", :align=> "middle", :style => "width:300px")%>
```
Comment (juanxcerv) [inline app/views/placeholder/landing.html.erb:37]: Maybe some padding on top of the bear and under the login button.

**Review by juanxcerv — APPROVED** [review]
Looks good to me. We will have to add some tests in the future. Also some styling decisions we can go over together/with LEAD.

File: app/views/placeholder/landing.html.erb (hunk 2 above)
Comment (melvinhernandez) [reply app/views/placeholder/landing.html.erb:11]: I agree with Juan, we should write css in a separate css file to for maintainability. But I think this is cool for this week's prototype.

File: app/views/placeholder/landing.html.erb (hunk 3 above)
Comment (melvinhernandez) [reply app/views/placeholder/landing.html.erb:37]: I think we should discuss how we're gonna be figuring out and deciding design feedback before we start our next iteration.

File: app/views/placeholder/landing.html.erb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+    </div>
+  </div>
+</div>
+
+
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```
Comment (melvinhernandez) [inline app/views/placeholder/landing.html.erb:53]: We should be taking care of adding jQuery and Bootstrap through the Gemfile instead
```
gem 'bootstrap'
gem 'jquery-rails'
```

File: app/controllers/placeholder_controller.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  public
+  
+  def landing
+    if session[:logged_in]
+      redirect_to root_path
+    end
+  end
+  
+  def login
+    session[:logged_in] = true
```
Comment (melvinhernandez) [inline app/controllers/placeholder_controller.rb:15]: This was a good idea!!! 👍

**Review by melvinhernandez — CHANGES_REQUESTED** [review]

**Pull request merged**